app/views/account/pages/_save.js.erb

```diff
@@ -0,0 +1,9 @@
+$("ul.errors").html("")
+<% if @page.errors.any? %>
+  <% @page.errors.full_messages.each do |message| %>
+    $("ul.errors").append($("<li />").html("<%= message.html_safe %>"))
+  <% end %>
+<% else %>
+  $("#testing").html("<%= escape_javascript(render @pages) %>")
```

Comment:
what is "#testing"?
why you rendering all pages here after error? WRONG

Reply:
what does it have to be instead of "#testing"?

Reply:
#pages, #pages_container, etc